src/Main.java

```diff
@@ -134,4 +134,22 @@ static void drawEqual(BufferedImage img, int offset) {
         g2d.fillRect(9 + offset, 89, 82, 2);
     }
 
+    static void EquationResult(String[] args) {
+        String result[] = new String[110];
+        int index = 0;
+        for (int i = 0; i <= 9; i++) {
+            for (int j = 0; j <= 9; j++) {
+                if (i + j < 10) {
+                    result[index++] = String.format("%d + %d = %d", i, j, i + j);

```

Comment:
Spaces between numbers and signs not needed.

Reply:
Done, change `result[index++] = String.format("%d + %d = %d", i, j, i + j)` to `result[index++] = String.format("%d-%d=%d", i, j, i - j)`